workflow/create_experiment.py

```diff
@@ -75,6 +77,12 @@ def input_args():
     data.update(os.environ)
     testconf = parse_j2yaml(path=user_inputs.yaml, data=data)
 
+    if 'exclude' in testconf:
```

Comment:
While `exclude` is fine, IMO, it is not descriptive in its intent.
The purpose here is to skip CI on the excluded host.
Would a key such as `skip_ci_on_hosts:` be descriptive to include in the yamls?
Thoughts?

Reply:
Oh yes completely agree!

Reply:
let me make the changes in a push